applications/nrf5340_audio/src/bluetooth/bt_management/scanning/bt_mgmt_scan.c

```diff
@@ -58,6 +72,12 @@ int bt_mgmt_scan_start(uint16_t scan_intvl, uint16_t scan_win, enum bt_mgmt_scan
 		return ret;
 	}
 
+	/* As the scanning is started before addr_print, and due to host API limitations,
```

Comment:
I dont completely understand this. It would be if we could have a quick walkthrough.
Why cant we do addr_print before starting to scan?

Reply:
Because the address is generated when scanning is started. I talked to Emil about this and requested some additions to the host API.